packages/core/core/src/utils/nuxt/_proxyUtils.ts

```diff
@@ -1,13 +1,35 @@
 import { Context as NuxtContext } from '@nuxt/types';
 import merge from 'lodash-es/merge';
 import { ApiClientMethod } from './../../types';
+import { AxiosInstance } from 'axios';
 
 interface CreateProxiedApiParams {
   givenApi: Record<string, ApiClientMethod>;
   client: any;
   tag: string;
 }
 
+const getUrl = (nxtContent: NuxtContext, endpoint: string) => {
```

Comment:
`middlewareUrl` is a full url and cannot be treated just like an `endpoint`. What if it comes like this: `https://yourdomain.com/api/`? 
The simplest thing to do is to just return `endpoint`, especially since it's overridden here:
https://github.com/vuestorefront/vue-storefront/pull/6688/files#diff-641a423be8f6d758deb84532fded11dd6e5173e5cddfbafef639e0be08622594R35

Reply:
I will update the PR with a fix on this